### PR TITLE
Feature/store data

### PR DESCRIPTION
**Pull request by francksefu**
Check the documentation for Ruby's [JSON library](https://ruby-doc.org/stdlib-3.0.0/libdoc/json/rdoc/JSON.html).
On exit from your app save all the data (books, people, and rentals) into JSON files. One per type of data you are storing and named accordingly (e.g. rentals.json) -> done
On startup of your app load all data from files. Remember to handle the case when one or more of the files are missing. -> done

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require_relative 'app'
+require_relative 'manage_data'
```
Comment (roshan-bajgain) [inline main.rb:3]: - [ ] While running `ruby main.rb` I am getting the error ``require_relative': cannot load such file --` what you can do is that please remove `require_relative 'manage_data'` form main.rb.

File: main.rb [hunk 2]
```diff
@@ -34,7 +35,10 @@ def main
   loop do
     options_menu
     choice = gets.chomp.to_i
-    exit if choice == 7
+    if choice == 7
+      app.save_people_data
+      exit
+    end
     handle_menu_choice(choice, app)
```
Comment (roshan-bajgain) [inline main.rb:42]: - [ ] While exiting the app I got the error shown below please fix this error and you are good to go. `the problem is app.save_people_data is undefined` please make sure that `On exit from the app the data should be saved (all the data.). For now, data is properly saving but make sure after fixing the issue is works as now.
### Error
Screenshot 2023-09-07 145257

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @francksefu and @HFG43, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- The data is saving ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @francksefu, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Highlights
- Great job, your project is error free :+1: 
- Nice work you meet all the requirements :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**